test/perfect_hashmap_test.cc

```diff
@@ -21,24 +21,22 @@ limitations under the License.
 #include "arrow/io/api.h"
 
 #include "basic/ds/array.h"
+#include "basic/ds/arrow.h"
+#include "basic/ds/arrow.vineyard.h"
 #include "basic/ds/hashmap.h"
 #include "client/client.h"
 #include "client/ds/object_meta.h"
+#include "common/util/arrow.h"
 #include "common/util/logging.h"
 
 using namespace vineyard;  // NOLINT(build/namespaces)
 
-int main(int argc, char** argv) {
-  if (argc < 2) {
-    LOG(INFO) << "usage ./perfect_hashmap_test <ipc_socket_name>";
-    return 1;
-  }
-  std::string ipc_socket = std::string(argv[1]);
-
+void test_double_id(std::string& ipc_socket) {
   Client client;
   VINEYARD_CHECK_OK(client.Connect(ipc_socket));
   LOG(INFO) << "Connected to IPCServer: " << ipc_socket;
 
+  // PerfectHashmapBuilder<int, double> builder(client);
```

Comment:
Remove if not used.

Please go through the PR by yourself as a double check.

Thanks!

Reply:
ok

Reply:
I will check for all similar issues